app/controllers/events_controller.rb

```diff
@@ -3,7 +3,6 @@ class EventsController < ApplicationController
 	def create
 		@team = Team.find_by(number: params[:team])
 		@competition = Competition.find_by(name: params[:competition]);
-		@goal = Goal.find_by(name: params[:goal]);
 		if @competition == nil || @team == nil || @goal == nil
```

Comment:
Noticed you removed `@goal` here but should also remove it from the if statement.